typescript/web/src/connectors/undo-store/effects/label-and-label-class.fixtures.ts

```diff
@@ -0,0 +1,140 @@
+import { FetchResult } from "@apollo/client";
+import { v4 as uuid } from "uuid";
+import { MATCH_ANY_PARAMETERS } from "wildcard-mock-link";
+import {
+  CreateLabelClassActionMutation,
+  CreateLabelClassActionMutationVariables,
+} from "../../../graphql-types/CreateLabelClassActionMutation";
+import {
+  DeleteLabelClassActionMutation,
+  DeleteLabelClassActionMutationVariables,
+} from "../../../graphql-types/DeleteLabelClassActionMutation";
+import {
+  GetLabelIdAndClassIdQuery,
+  GetLabelIdAndClassIdQueryVariables,
+} from "../../../graphql-types/GetLabelIdAndClassIdQuery";
+import {
+  UpdateLabelClassActionMutation,
+  UpdateLabelClassActionMutationVariables,
+} from "../../../graphql-types/UpdateLabelClassActionMutation";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../../../utils/tests/apollo-mock";
+import {
+  BASIC_LABEL_DATA,
+  DEEP_DATASET_WITH_CLASSES_DATA,
+} from "../../../utils/fixtures";
+import {
+  CREATE_LABEL_CLASS_QUERY,
+  DELETE_LABEL_CLASS_MUTATION,
+  GET_LABEL_QUERY,
+  UPDATE_LABEL_MUTATION as UPDATE_LABEL_MUTATION_SHARED,
+} from "./shared-queries";
+import { UPDATE_LABEL_MUTATION as UPDATE_LABEL_CLASS_OF_LABEL_MUTATION } from "./update-label-class-of-label";
+
+const [PREVIOUS_LABEL_CLASS_DATA] = DEEP_DATASET_WITH_CLASSES_DATA.labelClasses;
+
+export const GET_LABEL_ID_AND_CLASS_ID_MOCK: ApolloMockResponse<
+  GetLabelIdAndClassIdQuery,
+  GetLabelIdAndClassIdQueryVariables
+> = {
+  request: {
+    query: GET_LABEL_QUERY,
+    variables: { id: BASIC_LABEL_DATA.id },
+  },
+  result: {
+    data: {
+      label: {
+        id: BASIC_LABEL_DATA.id,
+        labelClass: { id: PREVIOUS_LABEL_CLASS_DATA.id },
+      },
+    },
+  },
+};
+
+const createLabelClassActionMockResult = jest.fn(
+  (
+    variables: CreateLabelClassActionMutationVariables
+  ): FetchResult<CreateLabelClassActionMutation> => ({
+    data: {
+      createLabelClass: {
+        id: variables.data.id ?? uuid(),
+        name: variables.data.name,
+        color: variables.data.color ?? "#123456",
```

Comment:
Fun-fact: `#123456` is a nice color :trollface: 
![image](https://user-images.githubusercontent.com/2091902/153055229-2e1dc213-be58-4d00-964b-1a1ecff20b7b.png)